interns-react-weekly-assignments/src/components/Home/Shreyansh/Week_4/Dashboard.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Routes,Route,Link } from "react-router-dom";
+import { DashboardWrapper } from "./DashboardStyle";
+import Feature1 from "./components/Feature1/Feature1";
+import Feature2 from "./components/Feature2/Feature2";
+import Feature3 from "./components/Feature3/Feature3";
+import Feature4 from "./components/Feature4/Feature4";
+import Feature5 from "./components/Feature5/Feature5";
+
+const Dashboard: React.FC = () => {
+  return (
+    <DashboardWrapper>
+      <div className="dashboard-container">
+          <nav className="navbar">
+            <ul className="feature-button">
```

Comment:
can you use a map for this instead of having multiple li elements also please add key for each li elements